ddtrace/tracer/option.go

```diff
@@ -138,19 +138,25 @@ func newConfig(opts ...StartOption) *config {
 		c.version = ver
 	}
 	if v := os.Getenv("DD_TAGS"); v != "" {
-		for _, tag := range strings.Split(v, ",") {
-			tag = strings.TrimSpace(tag)
+		sep, trim := " ", ","
```

Comment:
```suggestion
		sep  := " "
```
Let's drop that. I don't see any reason to trim commas, do you?